### PR TITLE
Add uglifyjs-folder dependency and create 'uglify' npm task.

### PR DESCRIPTION
**Pull request by omachala**
Add option to merge and minify output JS files using 'uglifyjs-folder' package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 TypeScript seed project. Use it to scaffold a TS project with all the tooling need to *properly* (opinated) set up a project.
 
-[![Build Status](https://travis-ci.org/metadevpro/ts-project-seed.svg?branch=master)](https://travis-ci.org/metadevpro/ts-project-seed)
+[![Travis](https://img.shields.io/travis/omachala/ts-project-seed.svg)](https://travis-ci.org/omachala/ts-project-seed)
```
Comment (pjmolina) [inline README.md]: Hi @omachala  
Review URLs to fix this pull-request. Should be pointing to the base project: in this case `metadevpro/ts-project-seed`.
This should be changed to local ones when instantiating the project-seed.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
...
+[![Coveralls github](https://img.shields.io/coveralls/github/omachala/ts-project-seed.svg)](https://coveralls.io/github/omachala/ts-project-seed)
```
Comment (pjmolina) [inline README.md]: Same here: url fix

**Review by pjmolina — COMMENTED** [review]
Thanks @omachala fro your contribution.
Before mergering, coveralls needs to be fixed.

**Comment by omachala** [comment]
@pjmolina yes you are right! I did some other changes after the pull-request was sent. So I changed shields url to original repo. Moreover it depends on you if you want to accept the other changes like adding 'uglifyjs-folder' and 'browserify'.

**Comment by pjmolina** [comment]
Thanks @omachala Merged!

**Pull request merged**